python/lsst/meas/algorithms/psfCandidate.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+
+from ._psfCandidate import PsfCandidateF
+
+__all__ = []  # import only for side effects
+
+PsfCandidateF.getCandidateRating = PsfCandidateF._getCandidateRating
```

Comment:
Indeed, now expanded to:

`SpatialCellCandidate.getCandidateRating` is defined in Python.
Therefore we cannot override it from the C++ wrapper.
Instead we give it a temporary name here, and assign it to the class from Python.